### PR TITLE
Flipp: use height value from server response

### PR DESCRIPTION
**Pull request by MishelleBit**
Using standardHeight and compactHeight from server response. Modified some tests accordingly as well.

### CODE REVIEW

**Comment by MishelleBit** [comment]
Hi, could I please get two reviews to proceed with this change? Thank you @przemkaczmarek @bsardo @onkarvhanumante @minaguib @dlackty @dmitris @asweeney86 @hhhjort

**Comment by MishelleBit** [comment]
Hi @przemkaczmarek, I hope you're doing well. Can we get an update on this?

**Review by przemkaczmarek — APPROVED** [review]

**Comment by MishelleBit** [comment]
Hello, could I please get another review to proceed with this change? Thank you @bsardo @onkarvhanumante @minaguib @dlackty @dmitris @asweeney86 @hhhjort

**Comment by MishelleBit** [comment]
Hi @bsardo , I hope you're doing well. Can we get an update on this?

**Comment by bsardo** [comment]
Hi @MishelleBit, we recently released PBS 3.0, more specifically v3.1.0, which updates Prebid Server package import references throughout the project from `v2` to `v3`.
For example:
```
import (
    "github.com/prebid/prebid-server/v3/adapters"
)
```
As a result, please merge with master (no rebase) and then ensure all Prebid Server package import references in the files you’ve changed are `v3` such that the test suite passes so we can resume reviewing. Thanks!

**Comment by MishelleBit** [comment]
Hi @bsardo, done!

**Comment by MishelleBit** [comment]
Hi @bsardo I hope you're doing well, can we get an update?

File: adapters/flipp/flipp.go [hunk 1]
```diff
@@ -231,7 +243,20 @@ func buildBid(decision *InlineModel, impId string) *openrtb2.Bid {
 		if decision.Contents[0].Data.Width != 0 {
 			bid.W = decision.Contents[0].Data.Width
 		}
-		bid.H = 0
+		customDataInterface := decision.Contents[0].Data.CustomData
+		customDataMap := customDataInterface.(map[string]interface{})
```
Comment (SyntaxNode) [inline adapters/flipp/flipp.go:247]: Please add a test case where customdata is a different type to verify this adapter code gracefully handles and error and doesn't panic on an incorrect response.

**Comment by MishelleBit** [comment]
Thanks @SyntaxNode ! It should handle different types and i've set `customeData` to null in `simple-banner-dtx.json`

**Review by przemkaczmarek — APPROVED** [review]

**Comment by MishelleBit** [comment]
Hi @przemkaczmarek can you please reapprove? thank you!

**Comment by MishelleBit** [comment]
Hi @SyntaxNode can you please reapprove? thank you!

**Review by przemkaczmarek — APPROVED** [review]

**Comment by MishelleBit** [comment]
@SyntaxNode Hello, you previously approved can you please re-approve? thank you.

**Review by VeronikaSolovei9 — APPROVED** [review]

**Review by przemkaczmarek — APPROVED** [review]

**Review by VeronikaSolovei9 — APPROVED** [review]

**Pull request merged**